### PR TITLE
Create firstbankpb.bank.xml

### PR DESCRIPTION
**Pull request by 6u621jpq6ybdw**

### CODE REVIEW

File: src/chrome/content/rules/firstbankpb.bank.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="firstbankpb.bank">
+	<target host="firstbankpb.bank" />
+	<target host="www.firstbankpb.bank" />
```
Comment (J0WI) [inline src/chrome/content/rules/firstbankpb.bank.xml]: `firstbankpb.com`
`www.firstbankpb.com`
`m.firstbankpb.com`
`olb.firstbankpb.com`

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

File: src/chrome/content/rules/firstbankpb.bank.xml [hunk 2]
```diff
@@ -1,7 +1,15 @@
+<!--
+	Invalid Certificate:
+		firstbankpb.com
+		www.firstbankpb.com
```
Comment (J0WI) [inline src/chrome/content/rules/firstbankpb.bank.xml]: `^` and `www` have an valid cert, but an incomplete chain.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

**Pull request merged**